src/components/text.rei

```diff
@@ -2,6 +2,7 @@ module type TextComponent = {
   let make:
     (
       ~accessible: bool=?,
+      ~accessibilityLabel: string=?,
```

Comment:
I would remove `accessibilityLabel` from the `Text` element, it didn't seem to work when I tested it. 
I think if you need it, you should wrap it with `View` and add `accessibilityLabel` on that instead.
Also, it's not mentioned in the [documentation](https://facebook.github.io/react-native/docs/accessibility.html)

Reply:
Nice catch! removed it removed in: ef88999